src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -120,6 +120,39 @@ public int RemovePackageReference(string projectPath, LibraryDependency libraryD
             }
         }
 
+        /// <summary>
+        /// Check if the project files format are correct for CPM
+        /// </summary>
+        /// <param name="packageReferenceArgs">Arguments used in the command</param>
+        /// <returns></returns>
+        public bool AreCentralVersionRequirementsSatisfied(PackageReferenceArgs packageReferenceArgs)
+        {
+            var project = GetProject(packageReferenceArgs.ProjectPath);
+
+            // Get package version and VersionOverride if it already exists in the props file. Returns null if there is no matching package version.
+            ProjectItem packageReferenceInProps = project.Items.LastOrDefault(i => i.ItemType == PACKAGE_REFERENCE_TYPE_TAG && i.EvaluatedInclude.Equals(packageReferenceArgs.PackageId));
+            var versionOverrideExists = packageReferenceInProps?.Metadata?.FirstOrDefault(i => i.Name.Equals("VersionOverride"));
```

Comment:
What would happen if I had:
```xml
<PackageReference Include="PackageA" VersionOverride="" />
```
I think this code should see if `!string.IsNullOrEmpty(versionOverride)`

Reply:
I think we could use the dg spec to get the information but in this case we are trying to verify that the project file and `Directory.Packages.props` are in a _good configuration_.